README.md

```diff
@@ -27,49 +27,59 @@ Algo VPN is a set of Ansible scripts that simplify the setup of a personal Wireg
 
-2. **[Download Algo](https://github.com/trailofbits/algo/archive/master.zip).** Unzip it in a convenient location on your local machine.
-
-3. **Install Algo's core dependencies.** Open the Terminal. `cd` into the `algo-master` directory where you unzipped Algo, then run:
-
-    - macOS:
-      ```bash
-      $ python3 -m ensurepip --user
-      $ python3 -m pip install --user --upgrade virtualenv
-      ```
-    - Linux (deb-based):
-      ```bash
-      $ sudo apt-get update && sudo apt-get install \
-          build-essential \
-          libssl-dev \
-          libffi-dev \
-          python3-dev \
-          python3-pip \
-          python3-setuptools \
-          python3-virtualenv -y
-      ```
-     - Linux (rpm-based): See the pre-installation documentation for [RedHat/CentOS 6.x](docs/deploy-from-redhat-centos6.md) or [Fedora](docs/deploy-from-fedora-workstation.md)
-     - Windows: See the [Windows documentation](docs/deploy-from-windows.md)
-
-4. **Install Algo's remaining dependencies.** Use the same Terminal window as the previous step and run:
+2. **Get a copy of Algo.** The Algo scripts will be installed on your local system. There are two ways to get a copy:
+
+	- Download the [ZIP file](https://github.com/trailofbits/algo/archive/master.zip). Unzip the file to create a directory named `algo-master` containing the Algo scripts.
+	
+	- Run the command `git clone https://github.com/trailofbits/algo.git` to create a directory named `algo` containing the Algo scripts.
+
+3. **Install Algo's core dependencies.** Algo requires that Python 3 and one supporting package are installed on your system.
+
+    - **macOS:** Apple does not provide Python 3 with macOS. There are two ways to obtain it:
+		* Use the [Homebrew](https://brew.sh) package manager. After installing Homebrew install Python 3 by running `brew install python3`.
+		
+		* Download and install the latest stable [Python 3 package](https://www.python.org/downloads/mac-osx/). Be sure to run the included *Install Certificates* command from Finder.
+		
+		Once Python 3 is installed on your Mac, from Terminal run:
+		```bash
+			python3 -m pip install --upgrade virtualenv
+		```
+
+    - **Linux:** Recent releases of Ubuntu, Debian, and Fedora come with Python 3 already installed. Make sure your system is up-to-date and install the needed supporting package:
+    	* Ubuntu and Debian:
+    	```bash
+    		sudo apt update && sudo apt upgrade -y
+    		sudo apt install -y python3-virtualenv
+    	```
+    	* Fedora:
+    	```bash
+    		sudo dnf upgrade -y
```

Comment:
I don't think that users need to upgrade the system before installing Algo, in some cases this may even screw up everything. So, let's just remove the upgrade step from both ubuntu and fedora steps